addons/website_sale/views/templates.xml

```diff
@@ -944,6 +958,27 @@
         </t>
     </template>
 
+    <template id="website_sale.filter_image_attributes" name="Image attributes in Filter">
+        <t t-foreach="a.value_ids" t-as="v">
+            <label
+                t-attf-class="css_attribute_image size_small position-relative rounded-3 bg-body text-center cursor-pointer #{'active' if v.id in attrib_set else ''}"
+            >
+                <input
+                    type="checkbox"
+                    name="attribute_value"
+                    t-att-value="'%s-%s' % (a.id, v.id)"
+                    t-att-checked="'checked' if v.id in attrib_set else None"
+                    t-att-title="v.name"
+                    class="w-100 h-100 opacity-0 cursor-pointer"
+                />
+                <div
+                    class="oe_img_bg o_bg_img_center position-absolute top-0 w-100 h-100 rounded-1"
+                    t-att-style="'background:url(/web/image/product.attribute.value/%s/image); background-size:cover;' % v.id if v.image else ''"
```

Comment:
Why use `background-image` in some places and `background` elsewhere? Is there a specific reason, or is it just a miss?
```suggestion
                    t-att-style="'background-image:url(/web/image/product.attribute.value/%s/image); background-size:cover;' % v.id if v.image else ''"
```